docs/code-quality/ca1801.md

```diff
@@ -35,6 +35,10 @@ This rule does not examine the following kinds of methods:
 
 - Methods used as event handlers.
 
+- Serialization constructors.
```

Comment:
Can we add hyperlink to base type/interface or some other generic documentation page about serialization constructors?